docs/api/middleware/session.md

```diff
@@ -16,12 +16,14 @@ This middleware uses our [Storage](https://github.com/gofiber/storage) package t
 func New(config ...Config) *Store
 func (s *Store) RegisterType(i interface{})
 func (s *Store) Get(c *fiber.Ctx) (*Session, error)
+func (s *Store) Delete(id string) error
```

Comment:
Isnt this from the other PR you submitted? It shouldnt be on this one.

Reply:
I have already rebased this PR and removed the commits from the other PR. So, it is ready for review and merge now.

Reply:
Thanks!